Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -247,6 +247,11 @@ bool ThisIsASceneObjectWithPrefabParent(out GameObject prefab)
                 return false;
             }
             prefab = (GameObject)PrefabUtility.GetCorrespondingObjectFromSource(gameObject);
+#elif UNITY_2018_2_OR_NEWER
+            PrefabType prefabType = PrefabUtility.GetPrefabType(gameObject);
+            if (prefabType == PrefabType.None)
+                return false;
+            prefab = (GameObject)PrefabUtility.GetCorrespondingObjectFromSource(gameObject);
```

Comment:
are you sure this didn't work in 2018.2 before?
unity did that in official hlapi: https://bitbucket.org/Unity-Technologies/networking/commits/5ae541db0cbf726f3b36b6c38deb50d67d9deb23#chg-Runtime/NetworkIdentity.cs

Reply:
That's what unity's api updater did